ehrapy/preprocessing/encoding/_encode.py

```diff
@@ -51,7 +51,8 @@ def encode(
         A dict containing the encoding mode and categorical name for the respective column (for each AnnData object in case of MuData object).
 
     Returns:
-        An :class:`~anndata.AnnData` object with the encoded values in X or None (in case of :class:`~mudata.MuData` object)
+        An :class:`~anndata.AnnData` object with the encoded values in X or None (in case of :class:`~mudata.MuData` object). The :class:`~anndata.AnnData`
```

Comment:
Don't think that this additional comment is required. This is the expected behavior I guess? Might be more confusing than helpful.